portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -103,7 +102,20 @@ function populateDropdown(list, ID) {
   });
 }
 
+// inserts a functioning searchbar into the navigation bar of a page.
```

Comment:
Tong's comments to address:
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450557971
Could you turn this into proper JSDoc? https://google.github.io/styleguide/jsguide.html#jsdoc-general-form.

Reply:
Yes I will adjust the comments accordingly